lib/watir-webdriver/user_editable.rb

```diff
@@ -23,10 +20,7 @@ def set(*args)
     #
 
     def append(*args)
```

Comment:
We can't alias because we have to require user_editable before elements/text_area & text_field